sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -353,18 +354,27 @@ public void setDsn(@Nullable String dsn) {
    * @return true if ON or false otherwise
    */
   public boolean isDebug() {
-    return debug;
+    return Boolean.TRUE.equals(debug);
   }
 
   /**
    * Sets the debug mode to ON or OFF Default is OFF
    *
    * @param debug true if ON or false otherwise
    */
-  public void setDebug(boolean debug) {
+  public void setDebug(final @Nullable Boolean debug) {
     this.debug = debug;
   }
 
+  /**
+   * Check if debug mode is ON, OFF or not set.
+   *
+   * @return true if ON or false otherwise
+   */
+  public @Nullable Boolean getDebug() {
```

Comment:
can we make it either private or package-private? `isDebug()` is all we need.

Reply:
Package private 👍

Reply:
.. or actually private would be better.